JS-Closures.md

```diff
@@ -1,6 +1,6 @@
 # JavaScript Closures
 
-Closures are functions that refer to independent (free) variables (variables that are used locally, but defined in an enclosing scope). In other words, the function defined in the closure 'remembers' the environment in which it was created.
+Closures are functions that refer to independent (free) variables. Free variables are variables that are used locally, but defined in an enclosing scope. In other words, the function defined in the closure 'remembers' the environment in which it was created.
```

Comment:
I'm reading Wikipedia now and it seems more like a closure is what's returned by `makeFunc()`:

> closure [refers to] a [function] whose open bindings (free variables) have been closed by (or bound in) the lexical environment, resulting in a closed expression, or closure.